build_container/build_container_ubuntu.sh

```diff
@@ -132,6 +132,55 @@ update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 1
 # TODO(phlax): use hashed requirements
 pip3 install -U pyyaml virtualenv
 
+function install_android_tools() {
+    #############
+    # Install JDK
+    #############
+
+    # Add Azul's public key
+    apt-key adv \
+        --keyserver hkp://keyserver.ubuntu.com:80 \
+        --recv-keys 0xB1998361219BD9C9
+
+    # Download and install the package that adds 
+    # the Azul APT repository to the list of sources 
+    curl -O https://cdn.azul.com/zulu/bin/zulu-repo_1.0.0-3_all.deb
+
+    # Install the Java 8 JDK
+    apt-get install -y ./zulu-repo_1.0.0-3_all.deb
+    apt-get update -y
+    apt-get install -y zulu8-jdk
+    rm ./zulu-repo_1.0.0-3_all.deb
+
+    #######################
+    # Install Android tools
+    #######################
+
+    sdk_install_target="/.android"
+    android_home="$sdk_install_target/sdk"
```

Comment:
i think rather use the vars from the env (and i guess add either add `$ANDROID_INSTALL_TARGET` or set the `sdk_install_target` as parent of `$ANDROID_HOME`)

it just means that its set one place authoritatively and less risk of it getting out of sync in some way

Reply:
Please let me know if I misunderstood your request, I tried to do what you were suggesting in 1a127b2399beae4181e5594f9d34b08ba36f1063.